src/openrct2/management/Marketing.cpp

```diff
@@ -46,16 +46,16 @@ int32_t marketing_get_campaign_guest_generation_probability(int32_t campaign)
     switch (campaign)
     {
         case ADVERTISING_CAMPAIGN_PARK_ENTRY_FREE:
-            if (park_get_entrance_fee() < 4)
+            if (park_get_entrance_fee() < MONEY(4,0))
                 probability /= 8;
             break;
         case ADVERTISING_CAMPAIGN_PARK_ENTRY_HALF_PRICE:
-            if (park_get_entrance_fee() < 6)
+            if (park_get_entrance_fee() < MONEY(6,0))
                 probability /= 8;
             break;
         case ADVERTISING_CAMPAIGN_RIDE_FREE:
             ride = get_ride(gMarketingCampaignRideIndex[campaign]);
-            if (ride->price < 3)
+            if (ride->price < MONEY(3,0))
```

Comment:
I'm not 100% sure if this occurrence should be modified as well. After all, rides can easily cost £0.30 if they're older, for example.

Reply:
Is there a way to check how this was set in vanilla RCT2?